tests/regression_tests/mg_temperature/test.py

```diff
@@ -0,0 +1,106 @@
+import os
+from tests.regression_tests.mg_temperature.build_2g import *
+from tests.testing_harness import *
+
+
+class MgTemperatureTestHarness(TestHarness):
+
+    def execute_test(self):
```

Comment:
I would prefer if the results_*.dat files were all aggregated together in a single file in tests/regression_tests/mg_temperature/ instead of located in the case subdirectories. Then, all of those case subdirectories can be completely removed after the test executes.